css/index.css

```diff
@@ -371,3 +371,30 @@ footer {
   font-size: 10px;
   text-align: center;
 }
+@media (max-width: 500px) {
```

Comment:
don't forget you can place media queries inside of each of the elements you want to change. It doesn't have to be done at the end.